superset-frontend/plugins/legacy-preset-chart-deckgl/package.json

```diff
@@ -24,10 +24,11 @@
     "lib"
   ],
   "dependencies": {
-    "@deck.gl/aggregation-layers": "^9.0.38",
-    "@deck.gl/core": "^9.0.37",
-    "@deck.gl/layers": "^9.0.38",
-    "@deck.gl/react": "^9.1.4",
+    "@deck.gl/aggregation-layers": "^9.1.9",
```

Comment:
nice side-win to upgrade to a newer deck.gl!

Reply:
wow btw looks like this ends up affecting 20k LoC in pacakge-lock.json (!?) curious on doing a sanity check on what's happening in there.... GitHub won't let me expand and don't have time to pull the branch to diff it, but would be great if you can validate/explain why changing a few lines here results in so much package-lock changes (?)

<img width="909" alt="Screenshot 2025-06-12 at 4 28 34 PM" src="https://github.com/user-attachments/assets/a9428474-9b46-49b1-b9ef-70bd31479e45" />

Reply:
looks much better now!